### PR TITLE
2039 ai branding

### PR DESCRIPTION
**Pull request by corykon**
image

image

image

image

### CODE REVIEW

File: src/app/foundations/ai/ai-demo.component.html
Comment (aaronbrethorst) [inline src/app/foundations/ai/ai-demo.component.html:2]: please replace all instances of ™ with `&#8482;`

File: src/app/foundations/ai/ai-demo.component.html
Comment (aaronbrethorst) [inline src/app/foundations/ai/ai-demo.component.html:12]: Replace

`"Oh! I know what this is, and it's going to be really useful. I love Healthcare.AI."`

with

`&quot;Oh! I know what this is, and it's going to be really useful. I love Healthcare.AI.&quot;`

**Review by aaronbrethorst — COMMENTED** [review]
here are the HTML validation issues I noticed.

**Comment by corykon** [comment]
I'm getting an error in the build complaining about the version of ng-packager. This is being updated in the ng14 PR anyways, so I'm going to wait until that goes in, merge it back, see if the error resolves itself

**Review by andrew-frueh — APPROVED** [review]
All good on the  checks once we merged the Angular 14 branch. So I'll go ahead and merge this too.

**Pull request merged**
